### DISCUSSION TITLE
is there a way to change the size of the white circle around your face  tracking?

### DISCUSSION THREAD

**Discussion by kingy000** [opentrack/opentrack#1846] [General]
I am using this with voice  attack. open track software for mouse movement with  WebCam  and  voice attack for mouse clicking and  keyboard
 I want to see if I can change the white  circle face tracking size because every time I  open my mouth to talk or use a command  from voice attack,   my  cursor keeps shaking and not staying steady.

**Comment by DaWelter** [comment]
You are talking about the neuralnet tracker, right?

**Reply by kingy000** [reply]
https://github.com/opentrack/opentrack/assets/38481599/f8833f49-96c8-48ad-ab3e-40637867dd4e

here is a video  of what I was saying
 as you can see the house is shaky

**Comment by DaWelter** [comment]
> I want to see if I can change the white circle face tracking size 

That's a misunderstanding. This circle shows the detected face / head size. It can't be changed. Basically the model is a black box which looks at a crop of the video feed and spits out the rotation, size, position and new bounding box. There is a checkbox in the options to turn on an overlay of the crop. The crop is based on the previous bounding box. Maybe that gives you a better idea.

> because every time I open my mouth to talk or use a command from voice attack, my cursor keeps shaking and not staying steady.

Yeah, that's a limitation. The model is just not good enough at outputting consistent poses. With the next update we can release new models which might do a little bit better. But I wouldn't count on serious improvements.

I have to say though that what you are trying to do is pretty hard code :-D And it's nice to see the tracker in action. Thanks for uploading the video. I recommend trying the point based tracker.

**Reply by DaWelter** [reply]
Btw. there is a "zoom" setting in the tracker options. It controls the size of the crop around the bounding box. I guess it's kinda what you're looking for. Try zooming in a bit more. Beware that there is certainly a sweet spot. Outside the 0.8 to 1.3 range the tracker will start having trouble.
